pkgs/development/tools/misc/texinfo/common.nix

```diff
@@ -40,7 +40,8 @@ stdenv.mkDerivation {
 
   # ncurses is required to build `makedoc'
   # this feature is introduced by the ./cross-tools-flags.patch
-  NATIVE_TOOLS_CFLAGS = if crossBuildTools then "-I${getDev buildPackages.ncurses}/include" else null;
+  NATIVE_TOOLS_CFLAGS = lib.optionals crossBuildTools [ "-I${getDev buildPackages.ncurses}/include" ]
+  ++ lib.optionals stdenv.isFreeBSD [ "-I${libintl}/include" ];
```

Comment:
How come it doesn't get added to the search path just by being added as an input?

Reply:
Oh that is very strange. It seems to be working fine without. Maybe this is a leftover from the impure bootstrap? I'll remove it now.